[2022-04-26 07:56:42,619] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: portfolio.download scheduled__2022-04-24T12:22:36.116809+00:00 [queued]>
[2022-04-26 07:56:42,646] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: portfolio.download scheduled__2022-04-24T12:22:36.116809+00:00 [queued]>
[2022-04-26 07:56:42,646] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 07:56:42,646] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-26 07:56:42,647] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 07:56:42,683] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): download> on 2022-04-24 12:22:36.116809+00:00
[2022-04-26 07:56:42,687] {standard_task_runner.py:52} INFO - Started process 627 to run task
[2022-04-26 07:56:42,696] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'portfolio', 'download', 'scheduled__2022-04-24T12:22:36.116809+00:00', '--job-id', '86', '--raw', '--subdir', '/usr/local/lib/python3.8/dist-packages/airflow/example_dags/portfolio.py', '--cfg-path', '/tmp/tmpfnsva978', '--error-file', '/tmp/tmpsspfs551']
[2022-04-26 07:56:42,697] {standard_task_runner.py:80} INFO - Job 86: Subtask download
[2022-04-26 07:56:42,870] {logging_mixin.py:109} INFO - Running <TaskInstance: portfolio.download scheduled__2022-04-24T12:22:36.116809+00:00 [running]> on host LAPTOP-FB8H3NSS.localdomain
[2022-04-26 07:56:43,051] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=portfolio
AIRFLOW_CTX_TASK_ID=download
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T12:22:36.116809+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-24T12:22:36.116809+00:00
[2022-04-26 07:56:43,053] {logging_mixin.py:109} INFO - Warning: Your Kaggle API key is readable by other users on this system! To fix this, you can run 'chmod 600 /root/.kaggle/kaggle.json'
[2022-04-26 08:05:53,795] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/dist-packages/airflow/models/taskinstance.py:1538 DtypeWarning: Columns (10) have mixed types. Specify dtype option on import or set low_memory=False.
[2022-04-26 08:05:53,853] {logging_mixin.py:109} INFO - 1045718
[2022-04-26 08:05:53,893] {logging_mixin.py:109} INFO -                                    belongs_to_collection    budget                                             genres  ...  video vote_average vote_count
adult                                                                                                                  ...                               
False  {'id': 10194, 'name': 'Toy Story Collection', ...  30000000  [{'id': 16, 'name': 'Animation'}, {'id': 35, '...  ...  False          7.7     5415.0
False                                                NaN  65000000  [{'id': 12, 'name': 'Adventure'}, {'id': 14, '...  ...  False          6.9     2413.0
False  {'id': 119050, 'name': 'Grumpy Old Men Collect...         0  [{'id': 10749, 'name': 'Romance'}, {'id': 35, ...  ...  False          6.5       92.0

[3 rows x 23 columns]
[2022-04-26 08:05:55,194] {python.py:175} INFO - Done. Returned value was: None
[2022-04-26 08:05:55,200] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=portfolio, task_id=download, execution_date=20220424T122236, start_date=20220425T225642, end_date=20220425T230555
[2022-04-26 08:05:55,254] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-26 08:05:55,281] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
